packages/common/src/ui/components/modals/ConfirmationModal/ConfirmationModalContext.ts

```diff
@@ -0,0 +1,39 @@
+import { createContext } from 'react';
```

Comment:
When I did the alert modal I found that separating the different pieces made it clearer. Hope it does for you also!

Reply:
yeah, it did. I wondered about splitting this one as well when I read through the alert modal